variables.tf

```diff
@@ -87,146 +119,176 @@ variable "worker" {
 }
 
 variable "rhel_image_name" {
+    type        = string
     description = "Name of the RHEL image that you want to use for the bastion node"
-    default = "rhel-8.2"
+    default     = "rhel-8.2"
 }
 
 variable "rhcos_image_name" {
+    type        = string
     description = "Name of the RHCOS image that you want to use for OCP nodes"
-    default = "rhcos-4.6"
+    default     = "rhcos-4.6"
 }
 
 variable "processor_type" {
+    type        = string
     description = "The type of processor mode (shared/dedicated)"
-    default = "shared"
+    default     = "shared"
 }
 
 variable "system_type" {
+    type        = string
     description = "The type of system (s922/e980)"
-    default = "s922"
+    default     = "s922"
 }
 
 variable "network_name" {
+    type        = string
     description = "The name of the network to be used for deploy operations"
-    default = "my_network_name"
+    default     = "my_network_name"
```

Comment:
can add validation check here as done for other variables non-empty and not equal to `my_network_name`

Reply:
The var.tfvars has value as "ocp-net". We can set the same here by default as we do for image names.

Reply:
@Elayaraja-Dhanapal please do this change

Reply:
@Prajyot-Parab Doing the changes

Reply:
@Prajyot-Parab @yussufsh changes done accordingly. Please validate. Thanks